website/docs/user_guide/projects.md

```diff
@@ -43,48 +43,46 @@ The configuration of a new Project is now available. the following input is avai
 
 ## Deleting an existing project {#deleting-an-existing-project}
 
-To keep your feature toggles clean, removing deprecated projects is important. From the overview of Projects – 
+To keep your feature toggles clean, removing deprecated projects is important. From the overview of Projects –
 1. In the top right of the project card, find the project menu represented by three vertical dots.
 
 
-![Project concept](/img/projects_menu_button.png)
+![A list of projects. Each project has three vertical dots — a kebab-menu — next to it.](/img/projects_menu_button.png)
```

Comment:
TIL - kebab-menu

I've heard hamburger for vertical, hadn't heard kebab :)

Reply:
There's also meatballs, bento, and döner. At least 😋 

![image](https://user-images.githubusercontent.com/17786332/146158635-7bbdeaf7-6bec-4d67-b1d8-6da5d8e44033.png)